api/proto/store/v1alpha2/object.proto

```diff
@@ -20,49 +17,38 @@ message ObjectRef {
 // objects in a content-addressable store.
 message Object {
   // Globally-unique content identifier of the object.
+  // Encodes fully-qualified type of the object as part of "codec".
   // Specs: https://github.com/multiformats/cid
   string cid = 1;
 
+  // Type of the object.
+  // Can be extracted from CID.
+  ObjectType type = 2;
```

Comment:
We already have a string `type` in OASF, please rename to `object_type` for example.